packages/virtualtours/src/react-components/NavBar.tsx

```diff
@@ -3,47 +3,47 @@ import { Link } from "react-router-dom";
 
 
 class NavBar extends React.Component {
+	private scrollTo = (id: string) => {
```

Comment:
When clicking on consultants, contacts, etc in landing page, will scroll down there (all are within single page).